Saving last configuration file to .certora_internal/24_05_10_19_34_48_207/run.conf
There is no TAC file. Going to script EVMVerifier/certoraBuild.py to main_with_args()
Creating dir /home/shan/my_project/test-audits/certora/contracts/.certora_internal/24_05_10_19_34_48_207/.certora_config
In /home/shan/my_project/test-audits/certora/contracts/token.spec, found the imports: []
copying spec file /home/shan/my_project/test-audits/certora/contracts/token.spec to /home/shan/my_project/test-audits/certora/contracts/.certora_internal/24_05_10_19_34_48_207/.certora_sources/.0_token.spec.spec
writing /home/shan/my_project/test-audits/certora/contracts/.certora_internal/24_05_10_19_34_48_207/.certora_verify.json
Path to typechecker is /home/shan/.local/lib/python3.10/site-packages/certora_jars/Typechecker.jar
running ['java', '-jar', '/home/shan/.local/lib/python3.10/site-packages/certora_jars/Typechecker.jar', '-buildDirectory', '/home/shan/my_project/test-audits/certora/contracts/.certora_internal/24_05_10_19_34_48_207']
Failed to compile spec files:
[main] ERROR ALWAYS - Found errors in token.spec:
[main] ERROR ALWAYS - Error in spec file (token.spec:6:35): Since CVL 2, methods block entries must end with `;`
[main] ERROR ALWAYS - Error in spec file (token.spec:6:51): Syntax error: unexpected token near `;`
build failed
Failure traceback: 
CertoraUserInputError: CVL specification syntax check failed
Encountered an error running Certora Prover:
CVL specification syntax check failed
